output/csv/config.go

```diff
@@ -76,11 +77,17 @@ func ParseArg(arg string) (Config, error) {
 		}
 		switch r[0] {
 		case "save_interval":
+			logger.Warnf("CSV output argument '%s' will soon be deprecated, please use 'saveInterval' instead.", r[0])
```

Comment:
```suggestion
			logger.Warnf("CSV output argument '%s' is deprecated, please use 'saveInterval' instead.", r[0])
```
It is deprecated, we just still support it. As I don't know if we will ever remove it (maybe when we redo configuration) I don't think it's needed to also add "we will remove it in v0.XX.0" or something like that that but it's possible I guess